src/components/TasksTable/index.jsx

```diff
@@ -155,77 +174,88 @@ export default class TasksTable extends Component {
     const iconSize = 14;
 
     return (
-      <div className={classes.tableWrapper}>
-        <DataTable
-          title="Bugs & Issues"
-          items={data}
-          renderRow={item => (
-            <TableRow tabIndex={-1} key={item.summary}>
-              <TableCell component="th" scope="row">
-                {item.project}
-              </TableCell>
-              <TableCell>
-                <List dense className={classes.summary}>
-                  <ListItem
-                    classes={{
-                      gutters: classes.summaryItem,
-                    }}
-                    button
-                    target="_blank"
-                    rel="noopener noreferrer"
-                    component="a"
-                    href={item.url}>
-                    <ListItemText>{item.summary}</ListItemText>
-                    <LinkIcon size={iconSize} />
-                  </ListItem>
-                </List>
-              </TableCell>
-              <TableCell className={classes.tags}>
-                {item.tags.map(tag => (
-                  <Chip
-                    name={tag}
-                    key={tag}
-                    label={tag}
-                    className={classNames({
-                      [classes.clickedChip]: tag === query.tag,
-                    })}
-                    onClick={this.handleTagClick}
-                  />
-                ))}
-              </TableCell>
-              <TableCell>{item.assignee}</TableCell>
-              <TableCell>
-                {formatDistance(item.lastUpdated, new Date(), {
-                  addSuffix: true,
-                })}
-              </TableCell>
-            </TableRow>
-          )}
-          headers={['Project', 'Summary', 'Tags', 'Assignee', 'Last Updated']}
-          sortByHeader={sortBy}
-          sortDirection={sortDirection}
-          onHeaderClick={this.handleHeaderClick}
-          filters={
-            showFilterContent && (
-              <div className={classes.filter}>
-                <TextField
-                  select
-                  label="Assignee"
-                  value={assignment}
-                  className={classes.dropdown}
-                  onChange={this.handleFilterChange}>
-                  {assignments.map(assignee => (
-                    <MenuItem key={assignee} value={assignee}>
-                      {assignee}
-                    </MenuItem>
+      <Fragment>
+        <div className={classes.flexContainer}>
+          <Button
+            variant="raised"
```

Comment:
`raised` puts too much emphasis on the button which we don't necessarily want to do here. Remove this line. We can use the default variant.